registrations/serializers.py

```diff
@@ -521,6 +527,84 @@ def update(self, instance, validated_data):
 
         return instance
 
+    def _validate_mandatory_fields(self, registration, validated_data, errors):
+        for field in registration.mandatory_fields:
+            if self.partial and field not in validated_data.keys():
+                # Don't validate field if request method is PATCH and field is missing from the payload.
+                continue
+            elif not validated_data.get(field):
```

Comment:
I think the old version `elif validated_data.get(field) in falsy_values` was a bit safer. If a new integer or boolean field needs to be added in the future, it is very easy to miss that this will start causing bugs.

Reply:
OK, missed these comments and went ahead and merged already... But this is a valid point so I'll do another PR to assess this and the other comment(s).